multiplayer/src/components/RemixGameButton.tsx

```diff
@@ -4,9 +4,10 @@ import { AppStateContext } from "../state/AppStateContext";
 
 export default function Render() {
     const { state } = useContext(AppStateContext);
+    const gameId = state.gameState?.gameId;
```

Comment:
```suggestion
    const { gameId } = gameState;
```

Reply:
This doesn't work since gameState can be undefined.